src/main.c

```diff
@@ -766,6 +799,12 @@ static int on_config_listen(h2o_configurator_command_t *cmd, h2o_configurator_co
     const char *hostname = NULL, *servname = NULL, *type = "tcp";
     yoml_t *ssl_node = NULL;
 
+#ifdef _WIN32
+    hostname = "127.0.0.1";
```

Comment:
On windows 0.0.0.0 is not a valid localhost or host name
So the default should be 127.0.0.1
If there is somewhere else in the code that can be changed to accommodate this we can move it :)